com.civicrm.hrjobroles/scss/hrjobroles.scss

```diff
@@ -17,3 +17,10 @@
     position: relative;
   }
 }
+
+
+// workaround to make sure that the texarea always go full-width
+// the !important is necessary to override the #bootstrap-theme style
+.editable-textarea {
+  display: block !important;
```

Comment:
no, cause the `#bootstrap-theme` shoreditch namespace, which contains the default style for xeditable, will still have a higher priority (because of the ID)